modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -112,6 +127,41 @@ a|
 
 New privilege that controls a user's access to desired configuration settings.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT name FOR ()-[r:TYPE]-() REQUIRE r.prop IS UNIQUE
+
+CREATE CONSTRAINT name FOR ()-[r:TYPE]-() REQUIRE r.prop IS RELATIONSHIP KEY
+----
```

Comment:
Is this not re-introduced in 5.7? It currently is under the 5.6 header?

Reply:
I thought I had checked that? but yes it should be 5.7